fifteenOdd.asm

```diff
@@ -0,0 +1,28 @@
+# COMP3410 Program Template
+# Author: Warner Johnson
+# Assignment: PA1
+# Date: 2/5/2015
+#
+# fifteenOdd.asm
+# Input: none
+# Output: sum of the odd integers between one and fifteen
+
+###############################################
+# .text segment. Assembly instructions go here.
+###############################################
+	.text
+	      li $s0, 1
+	      li $s2, 1
+	      li $s3, 15
+	    
+popOdd:
+	      addi $s2, $s2, 2
+	      add  $s0, $s0, $s2
+	      bne  $s2, $s3, popOdd
+	      
+	      la $a0, ($s0)
```

Comment:
-2 Line 23: This line works, but indirect addressing is an improper way to move a value to $a0.  By using ($s0) you're saying "load address of $s0", but $s0 is NOT an address, its a literal value.....It works but 'move' or 'add' would be more proper.